frontend/src/ui/time.tsx

```diff
@@ -9,6 +9,8 @@ type RelativeDateProps = {
 export const RelativeDate: React.FC<RelativeDateProps> = ({ date }) => {
     const { i18n } = useTranslation();
     const secsAgo = Math.floor((Date.now() - date.getTime()) / 1000);
+    const direction = secsAgo < 0 ? 1 : -1;
```

Comment:
JavaScript tip of the day:

```suggestion
    const direction = -Math.sign(secsAgo);
```